class.jetpack.php

```diff
@@ -4658,6 +4652,12 @@ public static function build_authorize_url( $redirect = false, $iframe = false )
 
 		$connection = self::connection();
 
+		$calypso_env = self::get_calypso_env();
+
+		if ( ! empty( $calypso_env ) ) {
+		    $args['calypso_env'] = $calypso_env;
```

Comment:
tabs > spaces :)

Reply:
This is addressed in https://github.com/Automattic/jetpack/pull/13369/commits/c5ef35ff263481674882958e7f22596967d7b2cf.